src/algebra/monoid_algebra/basic.lean

```diff
@@ -1503,6 +1533,18 @@ finset.induction_on s rfl $ λ a s has ih, by rw [prod_insert has, ih,
 
 end
 
+/--  An additive homomorphism `f : G →+ H` induces a `k`-algebra homomorphism
+`add_monoid_alg_hom_map k A f : add_monoid_algebra A G →ₐ[k] add_monoid_algebra A H`. -/
+def add_monoid_alg_hom_map (k A : Type*) [comm_semiring k] [semiring A] [algebra k A]
+  [add_monoid G] {H : Type*} [add_monoid H]
+  (f : G →+ H) : add_monoid_algebra A G →ₐ[k] add_monoid_algebra A H :=
```

Comment:
This should generalize to `add_monoid_hom_class` too

Reply:
Done!